.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
+  "singleQuote": true,
+  "trailingComma": "all",
+  "semi": false
```

Comment:
Please let's keep semis 😛

Reply:
@gaearon OK 😤

Reply:
If you really insist we can remove. You're the maintainer now 😄

Reply:
It's the fate of all my projects now. I pass them on and people remove the semicolons...

Reply:
It is not a real problem since Prettier does the job for us.

Reply:
Yea I know. Just funny.